### PR TITLE
squarewave

### PR DESCRIPTION
**Pull request by tokunbo**

### CODE REVIEW

**Comment by tokunbo** [comment]
It's for graph compare tests, I want the graph to look a very unique and exact way every time. :)

**Comment by glyph** [comment]
Normally I wouldn't say to rebase and force push something, but this commit appears to be by `MaybeToksMaybeNot <whatever@sanrio.co.jp>` which is not a real person :-).

Also, this is using an old version of the travis configuration, which makes it look like our builds are failing, and causes drastically longer build times, slowing down other PRs as well.

So please rebase this onto a recent master, with a correctly-configured git client that has your real github address in it, and force push.

**Comment by glyph** [comment]
I am applying WIP due to all the build failures - please attach "ready for review" label when it is :).

**Comment by tokunbo** [comment]
O_o; How do you see all that stuff? 

 I just see 1 .py file with 10 lines of changes. ...where's the old travis config?

**Comment by glyph** [comment]
@tokunbo I could see the build matrix in the travis build results :). and I could see the branch point in my local log viewer, which came from an old master.

**Comment by tokunbo** [comment]
@glyph  sounds like magic. I'll just copy this file and make a brand new branch from current master. Hopefully all the magic will be in alignment.

File: mimic/rest/maas_api.py [hunk 1]
```diff
@@ -276,7 +277,15 @@ def create_multiplot_from_metric(metric, reqargs, allchecks):
                     d = {}
                     d['numPoints'] = 4
                     d['timestamp'] = timestamp
-                    d['average'] = random.randint(1, 99)
+                    if not q % (points / 4):
+                        squarewave_downtrend = squarewave_downtrend ^ 1
+                    if c['label'] == 'squarewave':
```
Comment (glyph) [inline mimic/rest/maas_api.py]: This label is never "squarewave" in the tests, which means these lines of code are never tested.

**Comment by glyph** [comment]
@tokunbo - why'd you close? are you planning to test this some other way?

**Comment by glyph** [comment]
Oops didn't see #287

**Pull request closed without merging**